launchable/commands/subset.py

```diff
@@ -618,6 +650,9 @@ def run(self):
 
             # When Launchable returns an error, the cli skips showing summary
             # report
+            original_subset = subset_result.subset
+            original_rest = subset_result.rest
```

Comment:
This code is rather unnerving in so many ways:
* The name 'original_xyz' suggests this variable will be mutated later, and yet somehow we still need to maintain the distinction (which suggests leaky abstraction -- why are we needing to mutate this, and why do we need to tell them apart)?
* Yet the list is not copied, which suggests the mutation code will not just manipulate the list, but rather it clones or creates a whole new list.
* And yet no such mutation seems to be happening later.

I'm currently guessing this is just a refactoring remnant. If so, let's remove this. If not, please share with me the motivation behind this, so that I can work on the follow-up PR to do it differently.